src/content/getting-started/quickstart/external/_index.md

```diff
@@ -75,6 +75,24 @@ This guide assumes cluster-a's kubeconfig file is named `kubeconfig.cluster-a` a
 subctl deploy-broker --kubeconfig kubeconfig.cluster-a --globalnet
 ```
 
+#### Label gateway nodes
+
+When Submariner joins a cluster to the broker via the `subctl join` command, it chooses a node on which to install the 
+gateway by labeling it appropriately. By default, Submariner uses a worker node for the gateway; if there are no worker 
+nodes, then no gateway is installed unless a node is manually labeled as gateway. Since we are deploying k3s all-in-one 
+nodes, there are no worker nodes, so it is necessary to label the single nodes as gateway. By default, the node name is 
```

Comment:
```suggestion
nodes, then no gateway is installed unless a node is manually labeled as a gateway. Since we are deploying k3s all-in-one 
nodes, there are no worker nodes, so it is necessary to label the single node as a gateway. By default, the node name is 
```

Reply:
Done.